packages/dot-files/.cursor/rules/api-steps.mdc

```diff
@@ -1,63 +1,97 @@
 ---
-description: What are Motia API Steps and a Typescript example
+description: How to create basic API Steps in Motia
 globs: 
 alwaysApply: false
 ---
+
 # API Steps Guide
 
-API Steps expose HTTP endpoints for external interaction with workflows.
+API Steps expose HTTP endpoints that can trigger workflows and emit events.
 
-## Basic Configuration
+## Basic API Step Pattern
 ```typescript
-import { ApiMiddleware, ApiRouteConfig, ApiRequest, FlowContext, Handlers } from 'motia'
- 
-// Logging middleware
-const loggingMiddleware: ApiMiddleware = async (req, ctx, next) => {
-  ctx.logger.info('Request received', { path: req.pathParams })
-  const start = Date.now()
+import { ApiRouteConfig, Handlers } from 'motia'
+import { z } from 'zod'
+
+export const config: ApiRouteConfig = {
+  type: 'api',
+  name: 'create-user',
+  path: '/users',
+  method: 'POST',
+  emits: ['user.created'],
+  flows: ['user-management'],
+  bodySchema: z.object({
+    name: z.string(),
+    email: z.string().email()
+  })
+}
```

Comment:
Would it be helpful for AI to have comments similar to this file?

https://github.com/MotiaDev/motia/blob/main/packages/snap/src/create/templates/default/steps/01-api.step.ts.txt